### ISSUE TITLE
Stuck at random percentage of "Processing log"

### ISSUE THREAD

**Issue by LinusSte** [Manuel-777/MTG-Arena-Tool#73]
I keep encountering a strange bug, see title. 
First time I installed it yesterday, the tool was stuck somewhere around 20%. But as the tool worked after I reinstalled it, I didn't bother to submit an issue.
However, after rebooting I'm stuck again, and reinstalling does not fix the issue. This time, I get stuck at random percentages, most between 1% and 20%. Also if I close the app at this point, there still are 5 processes of it in my task manager that do not go away until I kill the task manually.
It can't be my log file, as other tools I use (MTGArena.Pro and MTGA Tracker) have no issues with it and work flawlessly.
v2.1.12, Windows 10

**Comment by Manuel-777** [comment]
Hi Linus! Other tools do not use the same parser as we do, as each has its own (and we probably do much more parsing and processing than any other!)

Please follow the troubleshooting guide to isolate the issue and let me know if it works! :)

**Comment by LinusSte** [comment]
Indeed, the issue was the output log. Unfortunately I cannot send you the old log file, as I had trouble attempting to rename and as that didn't work, delete the file. 
In both cases, Windows asked for Administrator permissions to do so, and despite having and granting those permissions I got an error message telling me that I do not have the required rights.
When I checked my running processes, I noticed there still were 5 phantom processes of MTGA Tool. After killing those, Windows deleted the logs.

I kinda think your application might have trouble with using Windows' Hibernation, it never got to catch up after waking the machine up, until I delete the logs. When I encounter this again I'll send you the logs.
